### ISSUE TITLE
[a11y] home page donate promo heading color contrast issue

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#14999]
## Description

The donate promo heading on the Mozilla home page has insufficient color contrast.

### Steps to reproduce

1. Open https://www.mozilla.org/en-US/
2. Scroll down to the donate banner

![image](https://github.com/user-attachments/assets/528a6309-5898-45d7-80c1-19035bbc9951)

### Expected result

The "Build our movement" heading should meet minimum color contrast ratio thresholds

### Actual result

![image](https://github.com/user-attachments/assets/8c812d2d-a75a-4025-9242-2d09c0d55be6)

### Environment

N/A

**Comment by janbrasna** [comment]
Tested that using `$color-moz-dark-purple` (`#6e008b`) improves that greatly at 8.05
(passes AAA level for any size text and AA for user interface components and graphical objects): https://webaim.org/resources/contrastchecker/?fcolor=6E008B&bcolor=E7DFFF

And it doesn't look too bad either, being more vibrant:

![Screenshot 2024-08-23 at 23 14 15](https://github.com/user-attachments/assets/012accca-dea0-48da-be7f-85967b08fd83)

**Comment by alexgibson** [comment]
@janbrasna that looks good to me, and +1 for choosing one of our existing tokens 👍
